### ISSUE TITLE
Modifications on App-bar

### ISSUE THREAD

**Issue by alibasir** [SaeedehTahmasebi66/chatter-app#1]
When we are going to put some HTML elements in a `div` and it is desired to **position** them relative to each other and also relative to their parent, it is needed to organize them in separated tags (like `span` or `div`) to give them `id`s or `class`es for later referencing and *styling*.
```html
<h1><img id="Moocus" src="https://ip.lfe.mw.tum.de/sections/moocus.png" alt="Moocus Image" style="height:56px;"> chatter|app</h1>
```
Here, you have a **dangled** text, i.e. chatter|app. It is required to put it in a `span` tag and give it an id for later styling (and positioning).
Then, put it in a `span` for example, and give it an `id` like **#app-name**.
Now, you have a parent element `h1` and two children `#Moocus` and `#app-name`.
Some styling may be like these (I only wrote **extra** properties):
```css
h1{
    position: relative;
}
#Moocus{
        position: absolute;
        left: 16px;
        height: 56px; /* the same as its parent height. It can also set like height: inherit; */
}

#app-name{
    position: absolute;
        /* We know that the width of Moocus is equal to its explicitly declared height, 56px,
         it has a 16px padding, so #app-name should be 72px far from the left side.
        */
    left: 72px; 
        
        /* We know that the height of h1 bar is 56px. By setting the height and line-height of #app-name to 
 the same value,
        we can align its text in the middle vertically.
        */
    height: 56px;
    line-height: 56px;
}
```

As you see, setting the height and line-height as the same height of its parent results in a vertical alignment of the text (here the app-name in the `h1`).

A more general approach can be setting the line-height of `h1`, not the children:
```css
h1{
    position: relative;
    line-height: 56px;
}
#app-name{
    position: absolute;
    left: 72px; 
    height: 56px;
 }

**Comment by alibasir** [comment]
I suggested two possible ways to set `line-height` properties for the App bar. 
As I checked your code, you mixed two ways!!!

**Comment by SaeedehTahmasebi66** [comment]
Yes... you are true.
At first, I did your first way for solving the issue. But I saw that there is another problem for my chat head, which has h1  tag,too. ( It was stick to the top). So I put  "line-height: 56px;"  for  h1  and forgot to delete it from #app-name {}. 
Now, I delete it from #app-name {} and also give the chat head a new id, #chat-head, for positioning it's left, too. (It doesn't have left position before).
